packages/entities/entities-keys/src/locales/en.json

```
@@ -70,6 +70,12 @@
           "tooltip": "An identifier for a key",
           "help": "If using JWK, need to match with the jwk.kid field"
         },
+        "x5t": {
+          "label": "X.509 Certificate Thumbprint (x5t)",
+          "placeholder": "Enter the certificate fingerprint that identifies the key to be used to validate JWT signature",
```

tiny nit: Shall we use a consistent glossary here?

```suggestion
          "placeholder": "Enter the certificate thumbprint that identifies the key to be used to validate JWT signature",
```

---

I will suggest this to veena, thank you for pointing out.